src/sveltekit/templates.ts

```diff
@@ -43,6 +44,13 @@ ${
   // Enable sending user PII (Personally Identifiable Information)
   // https://docs.sentry.io/platforms/javascript/guides/sveltekit/configuration/options/#sendDefaultPii
   sendDefaultPii: true,
+${
```

Comment:
ahh I guess that's where the newline is coming from (see my comments above). So we just need to ensure we don't print a newline here if !spotlightMode.